### ISSUE TITLE
[Zul'Aman] Amani Lynx Cub is invisible

### ISSUE THREAD

**Issue by Sn1krDudle** [chromiecraft/chromiecraft#7586]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

Generic

### Current Behaviour

Amani Lynx Cub mobs are invisible currently

### Expected Blizzlike Behaviour

Amani Lynx Cub should be visible and close to its 'parent' mob 
https://www.youtube.com/watch?v=googefOx9RA - 26:15 - 28:00

### Source

![zulaman-cub](https://github.com/user-attachments/assets/81d26eb2-367c-4bc8-b66e-2caeb1b4b71b)
![zulaman-cub-2](https://github.com/user-attachments/assets/11363fc4-12af-44d1-9a0a-41e07a97b61d)
![zulaman-cub-3](https://github.com/user-attachments/assets/bb1aa2c7-9d12-42ca-9803-b0d2b14393a2)

### Steps to reproduce the problem

1. Go to Zul'aman
2. Go to the mobs outside the boss room of Halazzi
3. Amani'shi Handlers and Amani Elder Lynx
4. Mobs should have Amani Lynx Cub mobs tethered to them and visible 

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/8c3c75122162c56ae3d73386206077ec23b7683e

### Operating system

Ubuntu 22.04

### Modules

- [mod-1v1-arena](https://github.com/azerothcore/mod-1v1-arena)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-anticheat](https://github.com/azerothcore/mod-anticheat)
- [mod-arena-3v3-solo-queue](https://github.com/azerothcore/mod-arena-3v3-solo-queue)
- [mod-arena-replay](https://github.com/azerothcore/mod-arena-replay)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-breaking-news-override](https://github.com/azerothcore/mod-breaking-news-override)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-costumes](https://github.com/azerothcore/mod-costumes)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-dmf-switch](https://github.com/azerothcore/mod-dmf-switch)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-npc-spectator](https://github.com/gozzim/mod-npc-spectator)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-eventScripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-ExchangeNpc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-RecruitAFriend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-SendAndBind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-TempAnnouncements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Sn1krDudle** [comment]
and sorry, i have no clue why my stupid pc included that first auda city picture with the screenshots :/

**Comment by amed80** [comment]
https://youtu.be/GNLXtBhnBYo?t=1725 28:45
https://youtu.be/lKlDl13T1O4?t=2516 41:55

[Amani Lynx Cub](https://wowgaming.altervista.org/aowow/?npc=24064) should be going in and out of stealth , currently they go into stealth and stay invisible until they get aggro

**Comment by TheSCREWEDSoftware** [comment]
@Sn1krDudle you have any source confirming that the behaviour is not intended?

Currently they are working as @amed80 showed. 

They are stealthed and they will un-stealth upon aggro and stealth again out of combat.

**Comment by amed80** [comment]
the bugged behavior for the lynx cub that currently out of combat they go into stealth indefinitely , they should be going in and out of stealth even outside of combat.

**Comment by TheSCREWEDSoftware** [comment]
I see what you mean now. that was my bad. But some of them seem to either don't come out of invisible (outside of combat, ever or they just have a long timer).

**Comment by TheSCREWEDSoftware** [comment]
Linked to AC: https://github.com/azerothcore/azerothcore-wotlk/issues/20857

**Comment by amed80** [comment]
I think there is 2 different mobs with stealth ability , [Amani Lynx Cub](https://wowgaming.altervista.org/aowow/?npc=24064) that should be going in and out of stealth and there is also [Amani Lynx](https://wowgaming.altervista.org/aowow/?npc=24043) which the ones that should be in stealth all the time until aggro'd but i believe atm [Amani Lynx](https://wowgaming.altervista.org/aowow/?npc=24043) are missing all their abilities in AC/CC

**Comment by TheSCREWEDSoftware** [comment]
In my case In the first video you showed. You can see many more Lynx Cubs which all appear when 1 Cub was aggro'd. But those seem to invisible (either longer or forever)

**Comment by amed80** [comment]
Yeah , not sure what are the timers for them to cast stealth or when they get out of it and it seems that they don't have the same timers

**Comment by Sn1krDudle** [comment]
I did some additional testing with the cubs. Hanging around near them for 30 min or so and none ever show visibly. Flares, detect invisibility and other abilities to 'see' invisi mobs doesnt work on them. I assume this is because they are not supposed to stay stealthed indefinitely so anything used to see them wont trigger an out of stealth flag for them.
